### ISSUE TITLE
Add setting to display theme images on NTP

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#34304]
## Description 

When users try to install custom themes, most of them will apply properly to the browser, but any theme images that would normally show on the NTP are not shown. Whatever setting the user has for the NTP in Brave will override this. 

Theme installed in Brave (w/"blank page" chosen):

<img width="1724" alt="image" src="https://github.com/brave/brave-browser/assets/3299098/be624d5f-b9cb-4a2d-8ad7-581dedfa035b">

Theme installed in Chrome:

![image](https://github.com/brave/brave-browser/assets/3299098/5a82f36a-e910-435d-b658-e8d7b23a31d5)

We now offer users to show the homepage, dashboard, a blank page, NTP Sponsored images, normal background images, show nothing (which results in colored gradient) or for users to upload and use their own NTP images. 

Since we appear to have a grasp on what is or isn't shown on the NTP, we should include an option for users to select `Theme image` when one is installed. This option would fit nicely in either of the following sections in `Settings`

![image](https://github.com/brave/brave-browser/assets/3299098/e2f974f6-96e4-4676-bbe5-72471e54d348)

![image](https://github.com/brave/brave-browser/assets/3299098/fa4b9079-8fe3-4e3e-9857-4ba96b443917)

Related issue:
https://github.com/brave/brave-browser/issues/29409

**Comment by Joli** [comment]
Yes please work on this! Would be a great change; I want to be able to customize my New Tab page and allow the background image for the theme I chose to show up. Was pretty disappointed it only changes the skin for tabs in Brave :/

**Comment by dodoo86** [comment]
cmon fix it pls

**Comment by Bmo4532** [comment]
Hope to see this feature fixed soon. Keep up the good work!

**Comment by 333HaI** [comment]
Lol bro its been 2 years, dont think theyre changing this...

**Comment by ilvipero** [comment]
It is still a nice to have feature but I am sure there are more pressing things to work on, worth keeping it in the backlog.

**Comment by Git-With-Chris** [comment]
+1

**Comment by 333HaI** [comment]
they dont care

**Comment by GingerBait** [comment]
I would love to see this inconvenience fixed.

**Comment by Brave-Matt** [comment]
+1s:
https://community.brave.app/t/desktop-brave-shows-gradient-background-when-wallpaper-is-disabled-android-works-correctly/648869

https://community.brave.app/t/this-is-also-my-problem/653760
